stormgls/stormgls.py

```diff
@@ -197,24 +205,26 @@ async def autocomplete(ls: StormLanguageServer, params: types.CompletionParams):
     atCursor = wordAtCursor(line, doc.lines[line], params.position.character)
 
     retn = []
+    depr = [types.CompletionItemTag.Deprecated,]
     if atCursor:
         word, rng = atCursor
         if word[0] == '$':
             for name, valu in ls.completions.get('libs', {}).items():
                 if name.startswith(word):
                     kind = types.CompletionItemKind.Property
                     if isinstance(valu.get('type'), dict):
-                        if valu['type'].get('type') == 'function':
+                        if valu['type'] == 'function':
```

Comment:
Something's not right here. How can `valu.get('type')` be a dict but `valu['type']` is also a string?

Reply:
Ah yea, something isn't quite right. The "something" is that I didn't finish this line. I needed another level of a "type" key dereference. Fixed and pushed :+1: